<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlightDialog</class>
 <widget class="QDialog" name="AddFlightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Flight</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 10pt;
}

QDialog {
	
	background-color: rgb(0, 0, 0);
}

QLabel {
	color: rgb(255, 246, 0);
}

QLineEdit, QComboBox, QDateTimeEdit, QSpinBox {
	color: rgb(255, 246, 0);
	
	background-color: rgb(0, 0, 0);
	border: 1px solid #fff600;
}

QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(255, 246, 0);
	border: 3px solid #fff600;
}

QPushButton:hover {
	background-color: #fff600;
	color: black;
}

#frame {
	
	background-color: rgb(255, 255, 255, 80);
	border-radius: 4px;
	padding: 5px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="lblAirlineName">
        <property name="text">
         <string>Airline:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAirline"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblFlightSerial">
        <property name="text">
         <string>Flight Serial:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFlightSerial"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblSource">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSource"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblDest">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDest"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lblDateTimeDeparture">
        <property name="text">
         <string>Date &amp; Time Departure:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dttmDeparture"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="lblNumPassengers">
        <property name="text">
         <string>Capacity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnPassengers"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lblNumHosts">
        <property name="text">
         <string>Number Of Hosts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnHosts"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lblDateTimeArrival">
        <property name="text">
         <string>Date &amp; Time Arrival:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dttmArrival"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btnSubmit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
